tensorflow/python/ops/rnn_cell_impl.py

```diff
@@ -701,11 +701,12 @@ def __init__(self, num_units,
       name: String, the name of the layer. Layers with the same name will
         share weights, but to avoid mistakes we require reuse=True in such
         cases.
+      dtype: The data type.
```

Comment:
Thanks @asimshankar for the review. The change of adding the `dtype` seems to have been introduced in a recent commit by other PR https://github.com/tensorflow/tensorflow/commit/eb35f19cf7e8c43cfb759bce2fab266ae753f0d0.

I have rebased and updated the PR so that only changes in the initializer part and the test cases remain.